libraries/ui-strings/build.gradle.kts

```diff
@@ -11,4 +11,8 @@ plugins {
 
 android {
     namespace = "io.element.android.libraries.ui.strings"
+
+    lint {
+        disable += "Typos"
+    }
```

Comment:
I would prefer that we fix the typos in Localazy when spotted. Was there some false positive?

Reply:
OK

Reply:
I'll address this in a separate PR, no worries.